### PR TITLE
Np 45735 treat nvi verified as regular verified

### PR DESCRIPTION
**Pull request by torarnet**

### CODE REVIEW

File: cristin-person/src/main/java/no/unit/nva/cristin/person/model/cristin/CristinPerson.java [hunk 1]
```diff
@@ -388,6 +391,21 @@ private Set<Employment> extractEmployments() {
...
+                              .map(PersonNvi::verifiedBy)
+                              .map(PersonSummary::id)
+                              .map(UriUtils::extractLastPathElement)
+                              .filter(Utils::isPositiveInteger)
+                              .isPresent();
+
+        if (nviVerified) {
+            return true;
+        } else {
+            return getIdentifiedCristinPerson();
```
Comment (monaullah) [inline cristin-person/src/main/java/no/unit/nva/cristin/person/model/cristin/CristinPerson.java:405]: Would it be less "expensive" to check `getIdentifiedCristinPerson()` first?

File: cristin-person/src/main/java/no/unit/nva/cristin/person/model/cristin/CristinPerson.java [hunk 2]
```diff
@@ -388,6 +391,21 @@ private Set<Employment> extractEmployments() {
                    .collect(Collectors.toSet());
     }
 
+    private Boolean extractVerified() {
+        var nviVerified = Optional.ofNullable(extractNvi())
```
Comment (monaullah) [inline cristin-person/src/main/java/no/unit/nva/cristin/person/model/cristin/CristinPerson.java:395]: Not sure if I understand why this part is necessary:
```
.map(PersonNvi::verifiedBy)
.map(PersonSummary::id)
.map(UriUtils::extractLastPathElement)
.filter(Utils::isPositiveInteger)
```

Comment (monaullah) [reply cristin-person/src/main/java/no/unit/nva/cristin/person/model/cristin/CristinPerson.java:395]: Would it be enough with `var nviVerified = Optional.ofNullable(extractNvi()).isPresent();`?
(I checked out your code and tried this, test was still green)

Comment (torarnet) [reply cristin-person/src/main/java/no/unit/nva/cristin/person/model/cristin/CristinPerson.java:395]: I just wanted to make sure we actually have data and not just empty objects, but maybe im overthinking it

File: cristin-person/src/main/java/no/unit/nva/cristin/person/model/cristin/CristinPerson.java (hunk 1 above)
Comment (torarnet) [reply cristin-person/src/main/java/no/unit/nva/cristin/person/model/cristin/CristinPerson.java:405]: Yes, i will change the order

Comment (torarnet) [reply cristin-person/src/main/java/no/unit/nva/cristin/person/model/cristin/CristinPerson.java:400]: Updated method

**Pull request merged**